cron/poll_read_issue_comments.py

```diff
@@ -145,6 +151,33 @@ def handle_vote_command(api, command, issue_id, comment_id, votes):
             gh.issues.close_issue(api, settings.URN, issue_id)
         elif sub_command == "reopen":
             gh.issues.open_issue(api, settings.URN, issue_id)
+        elif sub_command == "fast":
+            comment_updated_at = cmdmeta.comment.updated_at
+            comment_created_at = cmdmeta.comment.created_at
+            comment_poster = cmdmeta.comment.user
+            issue_poster = cmdmeta.issue.user
+            issue_created_at = cmdmeta.issue.created_at
+
+            # The post should not have been updated
+            if updated_at != created_at:
+                return
+
+            # The comment poster must be in the meritocracy
+            meritocracy = {}
+            with open('server/meritocracy.json', 'r') as mfp:
+                fs = fp.read()
+                if fs:
+                    meritocracy = json.loads(fs)
```

Comment:
`server/meritocracy.json` is temporary - it shouldn't ideally be read from because it's only there to fill the gap before the database is ready and we can use that instead.